### PR TITLE
Remove reference to internal PM

### PR DESCRIPTION
**Pull request by jtary**
Remove the references for the internal PM, and remove it’s package
requirement.

connect to #1596

### CODE REVIEW

**Comment by jtary** [comment]
@anthonyettinger PTAL

File: package.json [hunk 1]
```diff
@@ -47,7 +47,6 @@
     "strong-build": "^2.x",
     "strong-deploy": "^3.0.0",
     "strong-mesh-client": "^1.4.0",
-    "strong-pm": "^3.0.0",
```
Comment (sam-github) [inline package.json]: :clap: :clap: :clap: !!!!!!!!!!!!!!

**Comment by anthonyettinger** [comment]
LGTM

**Comment by jtary** [comment]
test please

**Pull request merged**